api/consensus.go

```diff
@@ -1,35 +1,31 @@
 package api
 
 import (
-	"fmt"
 	"net/http"
 
-	"github.com/NebulousLabs/Sia/build"
 	"github.com/NebulousLabs/Sia/types"
 )
 
-// The ConsensusGET struct contains general information about the consensus
-// set, with tags to support idiomatic json encodings.
+// ConsensusGET contains general information about the consensus set, with tags
+// to support idiomatic json encodings.
 type ConsensusGET struct {
 	Height       types.BlockHeight `json:"height"`
 	CurrentBlock types.BlockID     `json:"currentblock"`
 	Target       types.Target      `json:"target"`
 }
 
+// ConsensusBlockGET contains a block.
+type ConsensusBlockGET struct {
+	Block types.Block `json:"block"`
+}
```

Comment:
this seems a little odd. Why do you need a new type here? Or, if you do want a new type for consistency, why not embed the Block type?